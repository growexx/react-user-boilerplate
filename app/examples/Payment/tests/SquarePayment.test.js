import React from 'react';
import { render } from 'react-testing-library';

import SquarePayment from '../SquarePayment';

describe('<SquarePayment />', () => {
  it('should render with appId and location', () => {
    const props = {
      appId: '1234abc',
      location: 'aloc1234',
      amount: '100',
      requestPaymentSquare: () => {},
    };
    const {
      container: { firstChild },
    } = render(<SquarePayment {...props} />);
    expect(firstChild).toMatchSnapshot();
  });
});
