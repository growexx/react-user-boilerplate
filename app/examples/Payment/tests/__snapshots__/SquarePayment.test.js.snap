// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SquarePayment /> should render with appId and location 1`] = `null`;
